typegraph/python/typegraph/runtimes/deno.py

```diff
@@ -134,12 +134,15 @@ def __attrs_post_init__(self):
 
             from typegraph.graph.typegraph import get_absolute_path
 
-            path = get_absolute_path(self.file)
-            if os.environ.get("DONT_READ_EXTERNAL_TS_FILES"):
-                object.__setattr__(self, "code", f"file:{path}")
-            else:
-                with open(path) as f:
-                    object.__setattr__(self, "code", f.read())
+            rel_path = os.path.join("scripts", "deno", self.file)
+            path = get_absolute_path(rel_path)
+            object.__setattr__(self, "code", f"file:{path}")
+
+            # if os.environ.get("DONT_READ_EXTERNAL_TS_FILES"):
```

Comment:
@Natoandro can we remove this everywhere?

Reply:
Yes.